src/dmd/toobj.d

```diff
@@ -309,7 +309,8 @@ void toObjFile(Dsymbol ds, bool multiobj)
         override void visit(FuncDeclaration fd)
         {
             // in glue.c
-            FuncDeclaration_toObjFile(fd, multiobj);
+            if (fd.inlining != PINLINE.always)
```

Comment:
What if i want to always inline excepted that one time where i take its address as a function pointer ?

Reply:
If you take the address of an inline function, you emit it locally. You certainly don't extern to an inline...

Reply:
https://forum.dlang.org/post/mailman.6386.1547180284.29801.digitalmars-d-bugs@puremagic.com